cumulus_library/studies/vocab/vocab_icd_builder.py

```diff
@@ -17,19 +17,14 @@ def run_executor(self, cursor: object, schema: str, verbose: bool):
 
     @staticmethod
     def clean_row(row, filename):
-        """Removes non-SQL safe charatcers from the input row.
-
-        WARNING: This function, for the intended datasource specifically, drops the last
-        column since it is a known duplicate. This should be re-examined for other
-        datasources
-        """
-        for i in range(len(row) - 1):
+        """Removes non-SQL safe charatcers from the input row."""
+        for i in range(len(row)):
             cell = str(row[i]).replace("'", "").replace(";", ",")
             row[i] = cell
-        return row[:-1]
+        return row
 
     def create_icd_legend(
-        self, cursor: object, schema: str, verbose: bool, partition_size: int = 2000
+        self, cursor: object, schema: str, verbose: bool, partition_size: int = 1200
```

Comment:
this is becoming a very long running job. #60 would maybe make this less painful/expensive.

Reply:
#60 is this PR - is that what you meant?

Reply:
sigh - #57